pilot/pkg/model/context.go

```diff
@@ -408,6 +408,7 @@ func DefaultMeshConfig() meshconfig.MeshConfig {
 		ConnectTimeout:                    types.DurationProto(1 * time.Second),
 		IngressControllerMode:             meshconfig.MeshConfig_STRICT,
 		IngressService:                    "istio-ingressgateway",
+		IngressClass:                      "istio",
```

Comment:
@linsun You removed this in https://github.com/istio/istio/pull/11950/files#r259056328
Although not in the Helm chart seems like we should still have it to avoid the errors.
If you have no objection let's push that into `release-1.1` to fix the annoying errors.

Reply:
I think the problem here is because the ingress controller is enabled by default. What Lin did was to disable it. But she forgot to change the IngressControllerMode variable to off. Right now, it says its "strict", i.e. istio will still try to do leader election for ingress. If you set it to off, the errors would go away I assume?

Reply:
yes I removed it because the default mesh file we mounted to the pilot pod doesn't have this config.

Reply:
+1 we should disable leader election for ingress when ingress is off.

Reply:
@sbezverk you can just remove this line:

IngressControllerMode:             meshconfig.MeshConfig_STRICT,

BTW, this is the default file we should be using, and I forgot to remove the above line.
https://github.com/istio/istio/pull/11950#issuecomment-466117143